src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCli.java

```diff
@@ -432,6 +426,8 @@ public void logout() throws IOException {
     public List<ServiceInstance> getServices(String project, String application) {
         try {
             ServiceCatalogClient sc = client.adapt(ServiceCatalogClient.class);
+            client.services().inNamespace(project).withLabelSelector(new LabelSelectorBuilder().addToMatchLabels(KubernetesLabels.APP_LABEL, application).build()).list().getItems();
```

Comment:
Not sure to understand what is the purpose of this call ?

Reply:
good catch. i'll remove this.